connection.js

```diff
@@ -90,9 +91,13 @@ export async function connect(id, password, roomName) {
     let { jwt } = state['features/base/jwt'];
     const { iAmRecorder, iAmSipGateway } = state['features/base/config'];
 
-    if (!iAmRecorder && !iAmSipGateway && !jwt && isVpaasMeeting(state)) {
-        jwt = await getJaasJWT(state);
-        APP.store.dispatch(setJWT(jwt));
+    if (!iAmRecorder && !iAmSipGateway && isVpaasMeeting(state)) {
+        await APP.store.dispatch(getCustomerDetails());
```

Comment:
Mobile doesn't use this file, is this something that we need to do on mobile too?

Reply:
I'm thinking yes, but it wasn't in the scope of this PR, I just prepared the groundwork

Reply:
Ack